src/test/java/dog/pawbook/testutil/DatabaseBuilder.java

```diff
@@ -0,0 +1,34 @@
+package dog.pawbook.testutil;
+
+import dog.pawbook.model.Database;
+import dog.pawbook.model.managedentity.owner.Owner;
+
+/**
+ * A utility class to help with building Addressbook objects.
+ * Example usage: <br>
+ *     {@code AddressBook ab = new AddressBookBuilder().withOwner("John", "Doe").build();}
+ */
+public class DatabaseBuilder {
```

Comment:
probably need separate this rename as part of another commit so as to track as a rename instead of creation of new file. There's several references that needs to be renamed as well